backend/app/services/execution/agents/video/recovery.py

```
@@ -0,0 +1,418 @@
+        if not task_crd.spec.teamRef:
+            logger.warning(f"[video_recovery] No teamRef in task {task.id}")
+            return {}
+
+        # Find team
+        team = (
+            db.query(Kind)
+            .filter(
+                Kind.kind == "Team",
+                Kind.name == task_crd.spec.teamRef.name,
+                Kind.namespace == task_crd.spec.teamRef.namespace,
+                Kind.is_active == True,
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)